src/elements/content-sidebar/activity-feed/SelectableActivityCard.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../common/variables';
+
+@keyframes bcs-SelectableActivityCard-hover {
+    from {
+        background-color: transparent;
+    }
+
+    to {
+        background-color: $bdl-box-blue-05;
+    }
+}
+
+.bcs-SelectableActivityCard {
+    &[aria-disabled='false'] {
+        cursor: pointer;
+
+        &:hover,
+        &:focus {
+            outline: none;
+            box-shadow: 0 0 8px 0 rgba(114, 132, 142, .16);
```

Comment:
Do we have a variable for this color?

Reply:
Probably worth asking design if they want to add this box-shadow style to the design system options